packages/admin-ui/src/pages/packages/components/Info/ContainerList.tsx

```diff
@@ -65,7 +66,22 @@ export const ContainerList = ({ dnp }: { dnp: InstalledPackageData }) => {
           </span>
 
           {allContainersRunning ? (
-            <MdPauseCircleOutline onClick={() => onStartStop()} />
+            <MdPauseCircleOutline
+              onClick={() =>
+                dnp.dnpName === "wifi.dnp.dappnode.eth"
+                  ? confirm({
+                      title: `Disabling wifi service `,
+                      text: "You may loose access",
```

Comment:
The "You may loose access" was just a reference but the text should be a little bit nicer and communicate to an experienced user more meaning and context. Without being too long or detailed on the other hand. I should also start with a "Warning, ..." to convey why this popup exist in the first place